### ISSUE TITLE
18 Jan 2024 - A11y Theme Builder Meeting Minutes

### ISSUE THREAD

**Issue by aaronreed708** [finos/a11y-theme-builder#773]
## Date
2024-01-18 - 9:15am EST

## Zoom info
- Meeting link: https://zoom.us/j/91013899295?pwd=Qnh5UVN1aUZ4ak9tODgvMVNLNUg5Zz09
- Meeting id: 910 1389 9295
- Password: 123225
- Call-in: +1 929 205 6099 US (New York) (Check calendar for more options)

## Untracked attendees
- Fullname, Affiliation, (optional) GitHub username
- ...

## Meeting notices
- FINOS **Project leads** are responsible for observing the FINOS guidelines for [running project meetings](https://community.finos.org/docs/governance/meeting-procedures/). Project maintainers can find additional resources in the [FINOS Maintainers Cheatsheet](https://community.finos.org/docs/finos-maintainers-cheatsheet).

- **All participants** in FINOS project meetings are subject to the [LF Antitrust Policy](https://www.linuxfoundation.org/antitrust-policy/), the [FINOS Community Code of Conduct](https://community.finos.org/docs/governance/code-of-conduct) and all other [FINOS policies](https://community.finos.org/docs/governance/#policies). 

- FINOS meetings involve participation by industry competitors, and it is the intention of FINOS and the Linux Foundation to conduct all of its activities in accordance with applicable antitrust and competition laws. It is therefore extremely important that attendees adhere to meeting agendas, and be aware of, and not participate in, any activities that are prohibited under applicable US state, federal or foreign antitrust and competition laws. Please contact legal@finos.org with any questions.

- FINOS project meetings may be recorded for use solely by the FINOS team for administration purposes. In very limited instances, and with explicit approval, recordings may be made more widely available.

## Agenda
- [x] Convene & roll call (5mins)
- [x] Display [FINOS Antitrust Policy summary slide](https://community.finos.org/Compliance-Slides/Antitrust-Compliance-Slide.pdf) 
- [x] Review Meeting Notices (see above)
- [x] Approve past meeting minutes
- [x] Meetings can be scheduled on FINOS event calendar and get Zoom invitations sent by contacting help@finos.org, providing invitee email addresses and date, time and length of meeting
- [x] Summary of Community Working Group meeting (@PaulaPaul)
- [x] AOB, Q&A & Adjourn (5mins)

## Minutes

- Informed Paula of how to schedule FINOS meetings through help email
- Paula gave an update CWG meeting: Prep for event with Lise/Cody's presentation, Roadmap, collaboration with AnitaB/DFS in Q1.
- #397: Ongoing, Keith needs to be done first, need to get Keith and Lise together.
- #544: Prithviraj (not present) may be working on this.
- #686: Need to assign Yessel and Lise to review design.
- #541: Work is done, but wanted to assign Dorothy before I closed it.
- **How do we assign people to issues that are not in the org? I've seen Aaron be able to do it before.**
- #719: Paula will close soon.
- #629: When Keith is done, Lise says they can merge their codes together.
- #738: Done but needs breaking down into smaller tasks.
- #739: Lise, Paula and Cody are getting together and will share what they discussed with Aaron.
- #687: Paula to send out request.
- #685: Needs to be verified.
- #757: Starting later today.
- #756: To assign Lise, Paula, Cody, Yessel.
- #752: Lise finished, Yessel fine-tuning.
- #760: Lise experimenting.
- #729: chart.js working, other blockers.
- #759: Dependent on SDK work and molecules
- #755: High-Priority as it is an accessibility issue.
- #100: Duplicate of #656.
- Lise requests comments and give thoughts on new Issues that are opened.
 
## Decisions Made
- [ ] Decision 1
- [ ] Decision 2
- [ ] ...

## Action Items
- [ ] Action 1
- [ ] Action 2
- [ ] ...

**Comment by evangk6** [comment]
Evan K - DFS

**Comment by PaulaPaul** [comment]
Paula Paul @PaulaPaul - Greyshore

**Comment by codyzu** [comment]
Cody Zuschlag - NearForm - @codyzu

**Comment by evangk6** [comment]
Lise Noble - DFS

**Comment by evangk6** [comment]
@aaronreed708 - FYI I have notes and will make the updates from what we discussed and leave notes in the issue later after my appt..

EDIT: Done as of this evening.

**Comment by aaronreed708** [comment]
Minutes approved in today's meeting.  Closing.
